src/device/rsp/rsp_core.c

```diff
@@ -306,27 +304,45 @@ void do_SP_Task(struct rsp_core* sp)
         rsp.doRspCycles(0xffffffff);
         timed_section_end(TIMED_SECTION_AUDIO);
         sp->regs2[SP_PC_REG] |= save_pc;
+
+        sp_delay_time = 4000;
     }
     else
     {
         sp->regs2[SP_PC_REG] &= 0xfff;
         rsp.doRspCycles(0xffffffff);
         sp->regs2[SP_PC_REG] |= save_pc;
+
+        sp_delay_time = 0;
     }
 
     sp->rsp_task_locked = 0;
     if ((sp->regs[SP_STATUS_REG] & (SP_STATUS_HALT | SP_STATUS_BROKE)) == 0)
```

Comment:
@loganmc10 I noticed that this is triggered in two different situations:
1) RSP is locked in an idle loop
2) RSP set the MI_INTR_SP flag via an MTC0 instruction

I think only the second case should schedule an SP interrupt!
In the first case we should allow the CPU to run for some time without triggering an SP interrupt. Maybe we can create a new dummy interrupt type?

I'll try to propose an implementation based on this commit.